src/Request/BasicResolver.php

```diff
@@ -71,19 +71,22 @@ protected static function getRequestHeaders(array $server)
      * request resolver.
      *
      * @param array $server the server variables
-     * @param array $post   the post variables
      *
      * @return array|null
      */
-    protected static function getInputParams(array $server, array $post)
+    protected static function getInputParams(array $server)
     {
         static $result;
 
         if ($result !== null) {
             return $result ?: null;
         }
 
-        $result = $post ?: static::parseInput($server, static::readInput());
+        if (isset($server['REQUEST_METHOD']) && strtoupper($server['REQUEST_METHOD']) === 'GET') {
+            $result = $_GET;
```

Comment:
Access to superglobals was intentionally omitted from anything other than the resolve method. Perhaps put it back there, so that this method is not messing with global state?

Reply:
Does it matter if the method is only being called statically from within the `resolve` method?